lib/i18n/backend/active_record.rb

```diff
@@ -45,6 +45,26 @@ def store_translations(locale, data, options = {})
           end
         end
 
+        def reload!
+          @initialized = false
```

Comment:
Just one note here, maybe we don't need `@initialized`?
We can assume that `@translations = nil` means not initialized, because `Translation.to_hash` will always return a hash.
So
```ruby
def reload!
  @translations = nil
  self
end

def initialized?
  !@translations.nil?
end

def init_translations
  @translations = Translation.to_hash
end

def translations(do_init: false)
  # ...
end
```

What do you think?

Reply:
Looks nicer without it, I agree. Will change!